scripts/process/scaleStates.R

```diff
@@ -33,8 +37,10 @@ process.scaleStates <- function(viz){
     thisCat <- mutate(thisCat, scaleFactor = newArea / area)
 
     finalScaleFactors <- bind_rows(finalScaleFactors, thisCat)
+    catvalues[[cat]] = legend.area * magicState$value / magicState$area
```

Comment:
Can this be mutated into finalScaleFactors somehow (attributes)?  We don't have a good way of producing multiple outputs from a single function right now.